addons/l10n_ma/views/res_partner_views.xml

```diff
@@ -5,13 +5,13 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='company_registry']" position="attributes">
-                <attribute name="nolabel">1</attribute>
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="company_registry" string="ICE" invisible="country_code != 'MA'"></field>
```

Comment:
Is it really the best way ? Some other l10n display that field on res.partner's form view ; but they use conditions like invisible="'CZ' not in fiscal_country_codes" ; which means the field could essentially be displayed multiple times, because multiple countries enter the condition. Shouldn't we rather handle that with a computed field controlling whether or not the field needs to be shown, and make it use a function returning all the country codes for which we want it, which would simply be overridden in each l10n needing it to add the right country code ? 

(This is for master ; in stable, you can keep it the way you did it ; I'd still use fiscal_country_codes instead of direcly use country_code, to properly handle multivat, though)

Reply:
Hmm, from what I understand, the `company_registry` field is displayed for all countries where it can be used if it's distinct from the Tax ID. However, the importance of this field varies between countries. In some countries, it’s used extensively and needs to appear under the `VAT` field with a different label. 

If we introduce that computed field, it might result in the field not being displayed at all for other countries, which would defeat the purpose of having the field in the first place, wouldn't it?